demo/index.html

```diff
@@ -64,6 +64,11 @@ <h1 class="page-header">Scroller Examples</h1>
         Bottom visible (Adapter)
       </a>
     </li>
+    <li>
+      <a href="fixedRowHeight/fixedRowHeight.html">
```

Comment:
I added demo, after `npm start` it is available as `http://localhost:5005/fixedRowHeight/fixedRowHeight.html`